[2023-08-29T19:43:56.306+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-29T19:43:56.321+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-29T19:43:56.332+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-29T19:43:56.333+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-29T19:43:56.354+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-27 00:00:00+00:00
[2023-08-29T19:43:56.358+0000] {standard_task_runner.py:57} INFO - Started process 357 to run task
[2023-08-29T19:43:56.364+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp_kw4elsa']
[2023-08-29T19:43:56.367+0000] {standard_task_runner.py:85} INFO - Job 21: Subtask import_data_to_csv
[2023-08-29T19:43:56.415+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-29T19:43:56.503+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [running]> on host e69093e17de8
[2023-08-29T19:43:56.526+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-29T19:43:56.626+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-29T19:43:56.634+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-29T19:43:56.637+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py --date 2023-08-27']
[2023-08-29T19:43:56.651+0000] {subprocess.py:86} INFO - Output:
[2023-08-29T19:43:56.817+0000] {subprocess.py:93} INFO - python: can't open file '/Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py': [Errno 2] No such file or directory
[2023-08-29T19:43:56.821+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-08-29T19:43:56.846+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-08-29T19:43:56.854+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230827T000000, start_date=20230829T194356, end_date=20230829T194356
[2023-08-29T19:43:56.877+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 21 for task import_data_to_csv (Bash command failed. The command returned a non-zero exit code 2.; 357)
[2023-08-29T19:43:56.939+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-08-29T19:43:56.963+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T18:56:46.293+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:46.304+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T18:56:46.317+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T18:56:46.319+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T18:56:46.333+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-27 00:00:00+00:00
[2023-08-30T18:56:46.339+0000] {standard_task_runner.py:57} INFO - Started process 546 to run task
[2023-08-30T18:56:46.343+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp48osxjpr']
[2023-08-30T18:56:46.347+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask import_data_to_csv
[2023-08-30T18:56:46.373+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:46.421+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [running]> on host 5e7fbf8b7901
[2023-08-30T18:56:46.445+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:46.536+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T18:56:46.538+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T18:56:46.540+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py --date 2023-08-27']
[2023-08-30T18:56:46.556+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T18:56:46.694+0000] {subprocess.py:93} INFO - python: can't open file '/Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py': [Errno 2] No such file or directory
[2023-08-30T18:56:46.699+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-08-30T18:56:46.719+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-08-30T18:56:46.722+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230827T000000, start_date=20230830T185646, end_date=20230830T185646
[2023-08-30T18:56:46.740+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 13 for task import_data_to_csv (Bash command failed. The command returned a non-zero exit code 2.; 546)
[2023-08-30T18:56:46.766+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-08-30T18:56:46.791+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:20:00.546+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:20:00.563+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:20:00.573+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:20:00.574+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:20:00.587+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-27 00:00:00+00:00
[2023-08-30T19:20:00.592+0000] {standard_task_runner.py:57} INFO - Started process 267 to run task
[2023-08-30T19:20:00.597+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpdifhbaa2']
[2023-08-30T19:20:00.606+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask import_data_to_csv
[2023-08-30T19:20:00.636+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:20:00.694+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [running]> on host 884d86529f65
[2023-08-30T19:20:00.712+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:20:00.796+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T19:20:00.798+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:20:00.799+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python ./dags/data_ingestion.py --date 2023-08-27']
[2023-08-30T19:20:00.816+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:20:00.942+0000] {subprocess.py:93} INFO - python: can't open file './dags/data_ingestion.py': [Errno 2] No such file or directory
[2023-08-30T19:20:00.949+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-08-30T19:20:00.972+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-08-30T19:20:00.979+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230827T000000, start_date=20230830T192000, end_date=20230830T192000
[2023-08-30T19:20:01.012+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 10 for task import_data_to_csv (Bash command failed. The command returned a non-zero exit code 2.; 267)
[2023-08-30T19:20:01.056+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-08-30T19:20:01.085+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:31:37.204+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:37.220+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:31:37.229+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:31:37.230+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:31:37.248+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-27 00:00:00+00:00
[2023-08-30T19:31:37.254+0000] {standard_task_runner.py:57} INFO - Started process 265 to run task
[2023-08-30T19:31:37.258+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpdgk468zf']
[2023-08-30T19:31:37.264+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask import_data_to_csv
[2023-08-30T19:31:37.283+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:37.330+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [running]> on host f6bcf103ff79
[2023-08-30T19:31:37.347+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:37.438+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T19:31:37.442+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:31:37.445+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-27']
[2023-08-30T19:31:37.464+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:31:41.161+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:31:41.188+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230827T000000, start_date=20230830T193137, end_date=20230830T193141
[2023-08-30T19:31:41.212+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:31:41.233+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:41:53.570+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:53.585+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:41:53.593+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:41:53.597+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:41:53.621+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-27 00:00:00+00:00
[2023-08-30T19:41:53.627+0000] {standard_task_runner.py:57} INFO - Started process 329 to run task
[2023-08-30T19:41:53.638+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmptgod6uxw']
[2023-08-30T19:41:53.642+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask import_data_to_csv
[2023-08-30T19:41:53.665+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:53.723+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [running]> on host 81eeb4b7c8a9
[2023-08-30T19:41:53.755+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:53.846+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T19:41:53.849+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:41:53.850+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-27']
[2023-08-30T19:41:53.862+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:42:00.058+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:42:00.093+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230827T000000, start_date=20230830T194153, end_date=20230830T194200
[2023-08-30T19:42:00.146+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:42:00.180+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:50:53.713+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:53.727+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:50:53.740+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:50:53.742+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:50:53.760+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-27 00:00:00+00:00
[2023-08-30T19:50:53.774+0000] {standard_task_runner.py:57} INFO - Started process 305 to run task
[2023-08-30T19:50:53.786+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpdghx85wp']
[2023-08-30T19:50:53.792+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask import_data_to_csv
[2023-08-30T19:50:53.815+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:53.882+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [running]> on host 206ecb1dc8df
[2023-08-30T19:50:53.908+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:53.982+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T19:50:53.984+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:50:53.986+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-27']
[2023-08-30T19:50:53.998+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:51:01.642+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:51:01.732+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230827T000000, start_date=20230830T195053, end_date=20230830T195101
[2023-08-30T19:51:01.869+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:51:01.894+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:54:26.254+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:26.264+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:54:26.274+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:54:26.276+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:54:26.291+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-27 00:00:00+00:00
[2023-08-30T19:54:26.296+0000] {standard_task_runner.py:57} INFO - Started process 259 to run task
[2023-08-30T19:54:26.303+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp25r01feg']
[2023-08-30T19:54:26.309+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask import_data_to_csv
[2023-08-30T19:54:26.328+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:26.384+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [running]> on host 86f9e97d6c70
[2023-08-30T19:54:26.407+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:26.491+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T19:54:26.494+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:54:26.499+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-27']
[2023-08-30T19:54:26.524+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:54:30.526+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:54:30.567+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230827T000000, start_date=20230830T195426, end_date=20230830T195430
[2023-08-30T19:54:30.623+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:54:30.679+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:05:49.233+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:49.249+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T20:05:49.260+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T20:05:49.261+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:05:49.274+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-27 00:00:00+00:00
[2023-08-30T20:05:49.283+0000] {standard_task_runner.py:57} INFO - Started process 466 to run task
[2023-08-30T20:05:49.287+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmponuagzln']
[2023-08-30T20:05:49.290+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask import_data_to_csv
[2023-08-30T20:05:49.310+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:49.371+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [running]> on host 3938d7abd9ff
[2023-08-30T20:05:49.393+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:49.469+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T20:05:49.473+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:05:49.477+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-27']
[2023-08-30T20:05:49.493+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:05:56.536+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:05:56.624+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230827T000000, start_date=20230830T200549, end_date=20230830T200556
[2023-08-30T20:05:56.660+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:05:56.726+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:33:50.110+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:50.121+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T20:33:50.130+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T20:33:50.132+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:33:50.149+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-27 00:00:00+00:00
[2023-08-30T20:33:50.156+0000] {standard_task_runner.py:57} INFO - Started process 283 to run task
[2023-08-30T20:33:50.160+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp5slrn9yt']
[2023-08-30T20:33:50.173+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask import_data_to_csv
[2023-08-30T20:33:50.198+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:50.265+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [running]> on host 961f38e8363e
[2023-08-30T20:33:50.290+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:50.373+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T20:33:50.375+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:33:50.377+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-27']
[2023-08-30T20:33:50.390+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:33:56.210+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:33:56.283+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230827T000000, start_date=20230830T203350, end_date=20230830T203356
[2023-08-30T20:33:56.315+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:33:56.339+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:48:23.526+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:23.543+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T20:48:23.558+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T20:48:23.559+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:48:23.577+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-27 00:00:00+00:00
[2023-08-30T20:48:23.584+0000] {standard_task_runner.py:57} INFO - Started process 341 to run task
[2023-08-30T20:48:23.588+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpxt7g4xpi']
[2023-08-30T20:48:23.592+0000] {standard_task_runner.py:85} INFO - Job 15: Subtask import_data_to_csv
[2023-08-30T20:48:23.613+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:23.677+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [running]> on host 8f66711a2898
[2023-08-30T20:48:23.700+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:23.858+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T20:48:23.860+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:48:23.862+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-27']
[2023-08-30T20:48:23.890+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:48:30.694+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:48:30.743+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230827T000000, start_date=20230830T204823, end_date=20230830T204830
[2023-08-30T20:48:30.786+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:48:30.815+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:07:36.658+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:36.673+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T21:07:36.689+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T21:07:36.690+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:07:36.711+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-27 00:00:00+00:00
[2023-08-30T21:07:36.720+0000] {standard_task_runner.py:57} INFO - Started process 298 to run task
[2023-08-30T21:07:36.735+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpszskz10m']
[2023-08-30T21:07:36.740+0000] {standard_task_runner.py:85} INFO - Job 15: Subtask import_data_to_csv
[2023-08-30T21:07:36.808+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:36.930+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [running]> on host cbc745b7550b
[2023-08-30T21:07:36.945+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:37.096+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T21:07:37.099+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:07:37.100+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-27']
[2023-08-30T21:07:37.114+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:07:52.779+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:07:52.869+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230827T000000, start_date=20230830T210736, end_date=20230830T210752
[2023-08-30T21:07:52.944+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:07:52.976+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:17:04.212+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:17:04.233+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T21:17:04.244+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T21:17:04.247+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:17:04.267+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-27 00:00:00+00:00
[2023-08-30T21:17:04.279+0000] {standard_task_runner.py:57} INFO - Started process 355 to run task
[2023-08-30T21:17:04.291+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpd6zvb9od']
[2023-08-30T21:17:04.296+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask import_data_to_csv
[2023-08-30T21:17:04.323+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:17:04.384+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-27T00:00:00+00:00 [running]> on host 5b9f14bf20e0
[2023-08-30T21:17:04.402+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:17:04.490+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T21:17:04.492+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:17:04.494+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-27']
[2023-08-30T21:17:04.509+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:17:12.678+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:17:12.707+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230827T000000, start_date=20230830T211704, end_date=20230830T211712
[2023-08-30T21:17:12.768+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:17:12.826+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
